### PR TITLE
Update test + rename method

### PR DESCRIPTION
**Pull request by dzhyv**

### CODE REVIEW

File: src/main/java/com/ss/ita/google/ui/pages/SearchResultPage.java [hunk 1]
```diff
@@ -54,12 +55,13 @@ public String getResultLinkUrl(int numberResultLink) {
         return getSearchedResultLink(numberResultLink).getAttribute("href");
     }
 
-    public SearchResultPage openResultPage(int pageNumber) {
+    public SearchResultPage openSearchResultPageNumber(int pageNumber) {
```
Comment (vladdmytriv) [inline src/main/java/com/ss/ita/google/ui/pages/SearchResultPage.java:58]: dont use Number in name . U have openSearchResultPage with variable pageNumber. Its obvious.

File: src/main/java/com/ss/ita/google/ui/pages/SearchResultPage.java [hunk 2]
```diff
@@ -54,12 +55,13 @@ public String getResultLinkUrl(int numberResultLink) {
...
         $x(String.format("//a[@aria-label = 'Page %s']", pageNumber)).click();
         return this;
     }
 
-    public int getCurrentPageNumber() {
-        return Integer.parseInt($x("//td[text()]").getText());
+    public SearchResultPage shouldHavePageNumber(int number) {
+        $x("//td[text()]").shouldHave(text(Integer.toString(number)));
```
Comment (vladdmytriv) [inline src/main/java/com/ss/ita/google/ui/pages/SearchResultPage.java:64]: Maybe change type of variable number to String?

File: src/test/java/com/ss/ita/google/OpenSearchResultPageNineTest.java [hunk 3]
```diff
@@ -5,15 +5,14 @@
...
     public void verifyOpenPageNine() {
         int pageNumber = 9;
-        int currentPageNumber = new HomePage()
+        new HomePage()
                 .doSearch("funny kitten")
-                .openResultPage(pageNumber)
-                .getCurrentPageNumber();
-        Assert.assertEquals(currentPageNumber, pageNumber);
+                .openSearchResultPageNumber(pageNumber)
+                .shouldHavePageNumber(pageNumber);
```
Comment (vladdmytriv) [inline src/test/java/com/ss/ita/google/OpenSearchResultPageNineTest.java:16]: dont do it like method shouldHavePageNumber(pageNumber). Do it like assertion in test.

**Review by vladdmytriv — CHANGES_REQUESTED** [review]
change

**Review by vladdmytriv — APPROVED** [review]

**Review by Percifalll — APPROVED** [review]

**Pull request merged**